DatabaseBuddy.Core/Extender/NumericExtender.cs

```diff
@@ -31,6 +31,36 @@ public static long ToLongValue(this object Value)
             return ToLongValue(Value, 0);
         }
 
+        #region [ToDouble]
+        public static double ToDoubleValue(this double Value)
```

Comment:
This `ToDoubleValue` doesn't makes any sense since it just return the provided value.